__tests__/__system__/cli/discard/urimap/__snapshots__/cli.discard.urimap.system.test.ts.snap

```diff
@@ -0,0 +1,104 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`CICS discard urimap command should be able to display the help 1`] = `
+"
+ COMMAND NAME
+ ------------
+
+   urimap
+
+ DESCRIPTION
+ -----------
+
+   Discard a urimap from CICS.
+
+ USAGE
+ -----
+
+   zowe cics discard urimap <urimapName> [options]
+
+ POSITIONAL ARGUMENTS
+ --------------------
+
+   urimapName		 (string)
+
+      The name of the urimap to discard. The maximum length of the urimap name is
+      eight characters.
+
+ OPTIONS
+ -------
+
+   --region-name  (string)
+
+      The CICS region name from which to discard the urimap
+
+ CICS CONNECTION OPTIONS
+ -----------------------
+
+   --host  | -H (string)
+
+      The CICS server host name.
+
+   --port  | -P (number)
+
+      The CICS server port.
+
+      Default value: 443
+
+   --user  | -u (string)
+
+      Mainframe (CICS) user name, which can be the same as your TSO login.
+
+   --password  | --pw (string)
+
+      Mainframe (CICS) password, which can be the same as your TSO password.
+
+   --reject-unauthorized  | --ru (boolean)
+
+      Reject self-signed certificates.
+
+      Default value: true
+
+   --protocol  | -o (string)
+
+      Specifies CMCI protocol (http or https).
+
+      Default value: http
+      Allowed values: http, https
+
+ PROFILE OPTIONS
+ ---------------
+
+   --cics-profile  | --cics-p (string)
+
+      The name of a (cics) profile to load for this command execution.
+
+ GLOBAL OPTIONS
+ --------------
+
+   --response-format-json  | --rfj (boolean)
+
+      Produce JSON formatted data from a command
+
+   --help  | -h (boolean)
+
+      Display help text
+
+   --help-examples  (boolean)
+
+      Display examples for all the commands in a the group
+
+   --help-web  | --hw (boolean)
+
+      Display HTML help in browser
+
+ EXAMPLES
+ --------
+
+   - Discard a urimap named URIMAPA to the region named MYREGION
```

Comment:
Should the word "to" on this line be replaced with "from" ("discard from the region")?

Reply:
It should. This has been updated in the strings as well as the snapshot.